packages/google_mobile_maps/lib/src/controller.dart

```diff
@@ -0,0 +1,156 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of google_mobile_maps;
+
+final MethodChannel _channel =
+    const MethodChannel('plugins.flutter.io/google_mobile_maps')
+      ..invokeMethod('init');
+
+/// Controller for a single GoogleMaps instance.
+///
+/// Used for programmatically controlling a platform-specific
+/// GoogleMaps view, once it has been created and integrated
+/// into the Flutter application.
```

Comment:
"once it has been created and integrated into the Flutter application." => "once it has been initialized."?

Reply:
Done.